documentation/bolt_examples.md

```diff
@@ -123,3 +123,226 @@ To convert the `restart_service.ps1` script to a task, giving you the ability to
+
+**Before you begin** 
+
+- Install Bolt and Puppet Development Kit (PDK).
+- Ensure you have Powershell and Windows Remote Management (WinRM) access.
+
+In this example, you:
+
+- Build a project-specific configuration using a Bolt project directory and PDK.
+- Download module content from the Puppet Forge.
+- Write a Bolt plan to apply Puppet code and orchestrate the deployment of a package resource using the Chocolatey provider.
+
+### 1. Build a project-specific configuration using a Bolt project directory Bolt runs in the context of a project directory. This directory contains all of the configuration, code, and data loaded by Bolt.
+    
+1. Create a module using the PDK command `pdk new module puppet_choco_tap` 
+
+2. To make the module a Bolt project directory, add a `bolt.yaml` file. 
+
+3. Add the following configuration to the `bolt.yaml` file. 
+
+```
+Modulepath: "./modules/:~/.puppetlabs/bolt-code/modules:~/.puppetlabs/bolt-code/site-modules"
+```
+
+This means that the project is going to support itself by deploying all module dependencies to a modules folder in the project. Note that you don’t have to create the `./modules` folder.
+
+4. Create an inventory file to store information about your nodes. This is stored as `inventory.yaml` by default in the project directory. Add the following code: 
+
+```
+groups:
+  - name: windows
+    nodes:
+      - chocowin0.classroom.puppet.com
+      - chocowin1.classroom.puppet.com
+    config:
+      transport: winrm
+      winrm:
+        user: Administrator
+        Password: <ADD PASSWORD>
+```
+
+5. To make sure that your inventory is configured correctly and that you can connect to all the target nodes, run the following command from inside the project directory: 
+
+```
+bolt command run 'echo hi' --targets windows
+```
+
+**Note:** The `--targets windows` argument refers to the node group defined in the inventory file.
+
+You should get the following output:
+
+```
+Started on x.x.x.x...
+Started on localhost...
+Finished on localhost:
+  STDOUT:
+    hi
+Finished on 0.0.0.0:
+  STDOUT:
+    hi
+Finished on 127.0.0.1:
+  STDOUT:
+    hi
+Successful on 3 nodes: 0.0.0.0:20022,localhost
+Ran on 3 nodes in 0.20 seconds
+```
+
+### 2. Download the Chocolatey module and its dependencies from the Puppet Forge. 
+
+Bolt uses a `Puppetfile` to install module content from the Forge. A `Puppetfile` is a formatted text file that specifies the modules and data you want in each environment.
+
+1. Create a file named `Puppetfile` in the project directory — which describes the Forge content you want to install: 
+
+```     
+# Modules from the Puppet Forge
+mod 'puppetlabs-chocolatey', '4.1.0'
+## dependencies
+mod 'puppetlabs-stdlib', '4.13.1' #install latest
+ mod 'puppetlabs-powershell', '2.3.0'
+mod 'puppetlabs-registry', '2.1.0'
+        
+# Modules from Git
```

Comment:
I think this commented out code is more confusing than it is worth. can we link (maybe to r10k readme) on how to do this?

Reply:
I still think we should delete the commented out code. We can link to the reference docs instead.

Reply:
@clairecadman I agree, and I think this still needs to be addressed